ext/com_dotnet/com_com.c

```diff
@@ -219,6 +210,11 @@ PHP_METHOD(com, __construct)
 		if (info.pwszName) efree(info.pwszName);
 	}
 
+	zend_string_release(server_name);
```

Comment:
If server_name was provided as a string originally, we probably shouldn't be freeing it?

Reply:
Oops, thanks!